### PR TITLE
Update docker compose to version 3 syntax

### LINKED ISSUE #7: docker compose file update
The compose file in this project would not work for me. After much debugging and following https://docs.docker.com/compose/django/ I got something working.

Would a PR for a compose file using version 3 syntax be welcome?

### PR DESCRIPTION
**Pull request by morenoh149**
* update docker compose to version 3 syntax
* omit database default port
* use latest postgres
* expand run command for readability

closes https://github.com/skorokithakis/django-project-template/issues/7

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,19 +1,25 @@
-version: '2'
+version: '3'
 
 services:
   redis:
     image: redis
   db:
-    image: postgres:10
+    image: postgres
```
Comment (skorokithakis) [inline docker-compose.yml:7]: Why unpin Postgres here?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
     environment:
       POSTGRES_PASSWORD: password
-    ports:
-      - "5432:5432"
```
Comment (skorokithakis) [inline docker-compose.yml:11]: Why this removal?

File: docker-compose.yml (hunk 1 above)
Comment (morenoh149) [reply docker-compose.yml:7]: to make it closer to https://docs.docker.com/compose/django/

File: docker-compose.yml (hunk 2 above)
Comment (morenoh149) [reply docker-compose.yml:11]: to make it closer to https://docs.docker.com/compose/django/

**Comment by morenoh149** [comment]
the version 3 bump is the only critical change. I can revert the other lines if you prefer.

**Comment by skorokithakis** [comment]
Yes, the pinning of the Postgres version is because otherwise the database will change from under you unexpectedly and irreversibly after an upgrade, and the port is so you can connect with your tools externally. They're improvements.

**Comment by morenoh149** [comment]
I think the port exposed by the db image is 5432 by default. I used the yaml file in the quickstart and there was no need to specify it. Reverting.

**Comment by skorokithakis** [comment]
Great, sounds good!

**Review by skorokithakis — APPROVED** [review]

**Comment by skorokithakis** [comment]
Thank you!

**Pull request merged**